src/qibo/quantum_info/superoperator_transformations.py

```diff
@@ -11,6 +11,7 @@
 from qibo.gates.abstract import Gate
 from qibo.gates.gates import Unitary
 from qibo.gates.special import FusedGate
+from qibo.quantum_info.basis import _get_paulis, _normalization
 from qibo.quantum_info.linalg_operations import singular_value_decomposition
```

Comment:
Maybe these two functions could be moved to `quantum_info.quantum_info.py`.

```suggestion
from qibo.quantum_info.basis import _get_paulis, _normalization
```

Reply:
They are shared by all the backends, in the sense that we do not need to customize them for each backend. Thus, I would prefer to leave them out of ``quantum_info.quantum_info`` which contains the backend specific customizations. Maybe I can move them to utils, in case.

Reply:
I'm fine with `utils`